Build fails on JDK 15 due to maven bundle plugin error

Using JDK 15, the project build command `mvn clean install` fails with the following error:

```
[INFO] --- maven-bundle-plugin:4.2.0:manifest (bundle-manifest) @ openpdf ---
[ERROR] An internal error occurred
java.util.ConcurrentModificationException
    at java.util.TreeMap.callMappingFunctionWithCheck (TreeMap.java:742)
    at java.util.TreeMap.computeIfAbsent (TreeMap.java:596)
    at aQute.bnd.osgi.Jar.putResource (Jar.java:288)
    at aQute.bnd.osgi.Jar$1.visitFile (Jar.java:202)
    at aQute.bnd.osgi.Jar$1.visitFile (Jar.java:177)
```

Updating `maven-bundle-plugin` to it's latest release `5.1.1` is solving this issue as I tested locally.

And I notice that there are few other maven plugins that can be updated. Enabling [Dependabot](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/) for this project would automatically create PRs to update external dependencies such as maven plugins, libraries.

--- a/.github/dependabot.yml
+++ b/.github/dependabot.yml
@@ -1,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"

--- a/pom.xml
+++ b/pom.xml
@@ -42,22 +42,22 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-        <bouncycastle.bcpkix.version>1.65</bouncycastle.bcpkix.version>
-        <bouncycastle.bcprov.version>1.65.01</bouncycastle.bcprov.version>
+        <bouncycastle.bcpkix.version>1.66</bouncycastle.bcpkix.version>
+        <bouncycastle.bcprov.version>1.66</bouncycastle.bcprov.version>
 
-        <spotbugs.version>4.0.3</spotbugs.version>
+        <spotbugs.version>4.1.3</spotbugs.version>
         <hamcrest.version>2.2</hamcrest.version>
         <java-module-name>com.github.librepdf.openpdf.parent</java-module-name>
-        <jupiter.version>5.5.2</jupiter.version>
+        <jupiter.version>5.7.0</jupiter.version>
         <maven.clean.plugin.version>3.1.0</maven.clean.plugin.version>
         <maven.compiler.plugin.version>3.8.1</maven.compiler.plugin.version>
-        <maven.dependency.plugin.version>3.1.1</maven.dependency.plugin.version>
-        <maven.jar.plugin.version>3.1.2</maven.jar.plugin.version>
-        <maven.javadoc.plugin.version>3.0.1</maven.javadoc.plugin.version>
-        <maven.plugin.bundle.version>4.2.0</maven.plugin.bundle.version>
-        <maven.plugin.repository.version>2.3.1</maven.plugin.repository.version>
+        <maven.dependency.plugin.version>3.1.2</maven.dependency.plugin.version>
+        <maven.jar.plugin.version>3.2.0</maven.jar.plugin.version>
+        <maven.javadoc.plugin.version>3.2.0</maven.javadoc.plugin.version>
+        <maven.plugin.bundle.version>5.1.1</maven.plugin.bundle.version>
+        <maven.plugin.repository.version>2.4</maven.plugin.repository.version>
         <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
-        <maven.source.version>3.0.1</maven.source.version>
+        <maven.source.version>3.2.1</maven.source.version>
         <maven.surefire.plugin.version>2.22.1</maven.surefire.plugin.version>
         <maven.test.failure.ignore>false</maven.test.failure.ignore>
         <mockito.version>2.25.1</mockito.version>

